code/modules/reagents/chemistry/reagents/alcohol.dm

```diff
@@ -30,7 +30,7 @@
 			O.visible_message("<span class='warning'>It wasn't enough...</span>")
 
 /datum/reagent/consumable/ethanol/reaction_mob(mob/living/M, method=REAGENT_TOUCH, volume)//Splashing people with ethanol isn't quite as good as fuel.
-	if(method == REAGENT_TOUCH)
+	if(method == REAGENT_TOUCH || method == REAGENT_SPLASH)
```

Comment:
This is significant shotgun surgery, the same change being made to nearly every `/datum/reagent`. If the check ever has to change again, someone would need to go through all of these places again to make modifications. Could you either pull the check into a macro or pull it up into a proc on `/datum/reagent`? The latter would also make it easier to make exceptions to the rule just by overriding the proc in a subclass.

Reply:
errr... making proc like that?
```